07-routing-gossip.md

```diff
@@ -299,6 +299,25 @@ to be ordered in ascending order, unknown ones can be safely ignored.
 Future fields beyond `addresses` can still be added, optionally with
 padding within `addresses` if they require certain alignment.
 
+### Security Considerations for Node Aliases
+
+The node alias field is user-defined and has no cryptographic validation apart
```

Comment:
I don't fully understand the need to specify the cryptographic validation. You mean there is no way for others to manipulate on the fly?